dashboard/lib/tasks/seed_in_test.rake

```diff
@@ -1,3 +1,5 @@
 Rake::Task['db:test:prepare'].enhance do
+  ActiveRecord::Base.establish_connection('test')
   Rake::Task['seed:test'].invoke
+  ActiveRecord::Base.establish_connection(ENV['RAILS_ENV'])  
```

Comment:
not sure if I understand your question, but I think, yes? The intention is to temporarily connect to the test database while running the seed:test task and then reconnect to the "normal" database when done.